.formatter.exs

```diff
@@ -0,0 +1,69 @@
+[
+  export: [
+    locals_without_parens: [
+      # Phoenix.Channel
+      intercept: 1,
+
+      # Phoenix.Router
+      connect: 3,
+      connect: 4,
+      delete: 3,
+      delete: 4,
+      forward: 2,
+      forward: 3,
+      forward: 4,
+      get: 3,
+      get: 4,
+      head: 3,
+      head: 4,
+      match: 4,
+      match: 5,
+      options: 3,
+      options: 4,
+      patch: 3,
+      patch: 4,
+      pipe_through: 1,
+      plug: 1,
```

Comment:
Some of these functions belong to Plug, and they are currently exported by its [`.formatter.exs`](https://github.com/elixir-plug/plug/blob/master/.formatter.exs#L3-L21). So, I'm not sure if they should be exported here.

Since `:import_deps` cannot be used inside`:export`,  I would say the final users should be responsible of importing Plug and Phoenix formatters `import_deps: [:phoenix, :plug]`.

Reply:
The `import_deps` configuration could be included in the templates of the Phoenix installer :)